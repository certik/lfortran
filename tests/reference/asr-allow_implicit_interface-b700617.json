{
    "basename": "asr-allow_implicit_interface-b700617",
    "cmd": "lfortran --indent --show-asr --allow-implicit-interface --no-color {infile} -o {outfile}",
    "infile": "tests/allow_implicit_interface.f90",
    "infile_hash": "3d73fa8cc6b2a34329665b09295d1469f049597c7ab7f0c1039e4370",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-allow_implicit_interface-b700617.stderr",
    "stderr_hash": "c387edd6d4f498d8fc201994634cefffa6666ddf07c6072dd6dbdb38",
    "returncode": 1
}