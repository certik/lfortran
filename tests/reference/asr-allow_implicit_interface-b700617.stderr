Traceback (most recent call last):
[2m  Binary file "[0m[1m[35m$DIR/src/bin/lfortran[39m[0m[2m", in _start()[0m
[2m  File "[0m[1m[35m./csu/../csu/libc-start.c[39m[0m[2m", line 392, in __libc_start_main_impl()[0m
[2m  File "[0m[1m[35m./csu/../sysdeps/nptl/libc_start_call_main.h[39m[0m[2m", line 58, in __libc_start_call_main()[0m
[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 1647, in ??[0m
    return emit_asr(arg_file, lfortran_pass_manager,
[2m  File "[0m[1m[35m$DIR/src/bin/lfortran.cpp[39m[0m[2m", line 552, in ??[0m
    r = fe.get_asr2(input, lm, diagnostics);
[2m  File "[0m[1m[35m$DIR/src/lfortran/fortran_evaluator.cpp[39m[0m[2m", line 225, in LFortran::FortranEvaluator::get_asr2(std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const&, LFortran::LocationManager&, LFortran::diag::Diagnostics&)[0m
    Result<ASR::TranslationUnit_t*> res2 = get_asr3(*ast, diagnostics);
[2m  File "[0m[1m[35m$DIR/src/lfortran/fortran_evaluator.cpp[39m[0m[2m", line 247, in LFortran::FortranEvaluator::get_asr3(LFortran::AST::TranslationUnit_t&, LFortran::diag::Diagnostics&)[0m
    compiler_options.symtab_only, compiler_options);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_to_asr.cpp[39m[0m[2m", line 53, in LFortran::ast_to_asr(Allocator&, LFortran::AST::TranslationUnit_t&, LFortran::diag::Diagnostics&, LFortran::SymbolTable*, bool, LFortran::CompilerOptions&)[0m
    auto res = body_visitor(al, ast, diagnostics, unit, compiler_options, template_type_parameters);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_body_visitor.cpp[39m[0m[2m", line 1740, in LFortran::body_visitor(Allocator&, LFortran::AST::TranslationUnit_t&, LFortran::diag::Diagnostics&, LFortran::ASR::asr_t*, LFortran::CompilerOptions&, std::map<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::vector<LFortran::ASR::asr_t*, std::allocator<LFortran::ASR::asr_t*> >, std::less<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > >, std::allocator<std::pair<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > const, std::vector<LFortran::ASR::asr_t*, std::allocator<LFortran::ASR::asr_t*> > > > >&)[0m
    b.visit_TranslationUnit(ast);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_body_visitor.cpp[39m[0m[2m", line 106, in LFortran::BodyVisitor::visit_TranslationUnit(LFortran::AST::TranslationUnit_t const&)[0m
    visit_ast(*x.m_items[i]);
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4645, in LFortran::AST::BaseVisitor<LFortran::BodyVisitor>::visit_ast(LFortran::AST::ast_t const&)[0m
    void visit_ast(const ast_t &b) { visit_ast_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4605, in ??[0m
    case astType::program_unit: { v.visit_program_unit((const program_unit_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4653, in LFortran::AST::BaseVisitor<LFortran::BodyVisitor>::visit_program_unit(LFortran::AST::program_unit_t const&)[0m
    void visit_program_unit(const program_unit_t &b) { visit_program_unit_t(b, self()); }
[2m  File "[0m[1m[35m$DIR/src/lfortran/ast.h[39m[0m[2m", line 4268, in ??[0m
    case program_unitType::Function: { v.visit_Function((const Function_t &)x); return; }
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_body_visitor.cpp[39m[0m[2m", line 1004, in LFortran::BodyVisitor::visit_Function(LFortran::AST::Function_t const&)[0m
    transform_stmts(body, x.n_body, x.m_body);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_body_visitor.cpp[39m[0m[2m", line 87, in LFortran::BodyVisitor::transform_stmts(LFortran::Vec<LFortran::ASR::stmt_t*>&, unsigned long, LFortran::AST::stmt_t**)[0m
    ASR::stmt_t* impl_decl = create_implicit_deallocate_subrout_call(tmp_stmt);
[2m  File "[0m[1m[35m$DIR/src/lfortran/semantics/ast_body_visitor.cpp[39m[0m[2m", line 942, in LFortran::BodyVisitor::create_implicit_deallocate_subrout_call(LFortran::ASR::stmt_t*)[0m
    const ASR::Var_t* orig_arg_var = ASR::down_cast<ASR::Var_t>(subrout->m_args[i]);
[2m  Binary file "[0m[1m[35m/lib/x86_64-linux-gnu/libc.so.6[39m[0m[2m", in __GI___sigaction()[0m
Segfault: Signal SIGSEGV (segmentation fault) received
